### PR TITLE
Migrate to slack-machine

### PR DESCRIPTION
**Pull request by Sarthak5598**

### CODE REVIEW

**Review by arkid15r — CHANGES_REQUESTED** [review]

**Comment by Sarthak5598** [comment]
Thank you @arkid15r for the suggestions

File: manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - command: /repo
+      description: Show repo based on technologies
+      should_escape: false
+    - command: /project
+      description: Gives links to all the projects
+      should_escape: false
+oauth_config:
+  scopes:
+    bot:
+      - app_mentions:read
```
[the remark this answers is not in the document]
Comment (Sarthak5598) [reply manifest.yaml]: changed it thank you!

File: manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      should_escape: false
+oauth_config:
+  scopes:
+    bot:
+      - commands
+      - users:read
+      - channels:read
+      - groups:read
+      - mpim:read
+      - im:read
```
Comment (arkid15r) [inline manifest.yaml:25]: Just sorted it:
```suggestion
      - channels:read
      - commands
      - groups:read
      - im:read
      - mpim:read
      - users:read
```

**Review by arkid15r — APPROVED** [review]
LGTM

**Comment by Sarthak5598** [comment]
Thank you @arkid15r

**Pull request merged**